crates/bitwarden-crypto/src/keys/pin_key.rs

```diff
@@ -19,6 +19,8 @@ impl PinKey {
     }
 
     /// Encrypt the users user key
+    ///
+    /// @param user_key: The user key to encrypt
```

Comment:
This is jsdoc's way of documenting function arguments, but rust docs doesn't have any special behavior with this, so it ends up looking super weird.

The standard library seems to just not document parameters in a list-form and lets the rest of the documentation and name imply the function of the parameter. Indeed, that feels totally sufficient here. If you find a place that's _not_ sufficient, let's figure something out, but I don't want to default to jsdoc format

Reply:
Ok, removed this for now.

Reply:
The most common way of documenting parameters in rust is like this, that said I agree that it's not super common to document them, specially if the name is obvious enough and there's no special or unexpected behavior.

https://github.com/bitwarden/sdk-internal/blob/5cb4eb03ed7ccc0c4ecc0875f78f718c8a2175e5/crates/bitwarden-crypto/src/store/context.rs#L133-L139